hack/tools.mk

```diff
@@ -59,7 +59,7 @@ YQ                         := $(TOOLS_BIN_DIR)/yq
 DOCFORGE_VERSION ?= v0.33.0
 GOLANGCI_LINT_VERSION ?= v1.50.1
 GO_APIDIFF_VERSION ?= v0.5.0
-GOIMPORTSREVISER_VERSION ?= v3.3.0
+GOIMPORTSREVISER_VERSION ?= 32c80678d5d73a50b6966f06b346de58b1d018f1
```

Comment:
With this change I believe we no longer need to call `make format` in `make generate`. This was introduced in https://github.com/gardener/gardener/pull/7349.
If that is the case, we also need to add a release note as currently all the extensions have this `make format` call in their `make generate`.